### DISCUSSION TITLE
Has anyone already tried to implement survey.js inside Quasar?

### DISCUSSION THREAD

**Discussion by Alfredoarnoldo** [quasarframework/quasar#16385] [Quasar App Extensions] [answered]
Using Survey.js inside vue 3 is working perfectly well but it seems much more difficult using quasar.
Using a boot file like this does not seem enough to load the component insite quasar vue files.

_import { surveyPlugin } from 'survey-vue3-ui'
import { boot } from 'quasar/wrappers'

export default boot(({ app }) => {
  app.use(surveyPlugin)
})_

Has someone already succeded in loading a basic survey from survey.js inside quasar? Otherwise is there any survey creator component and viewer available as a plugin a quasar?

Many thanks!

**Comment by yusufkandemir** [comment, marked as answer]
Your boot file contents look correct. You most probably forgot to add the boot file entry to `quasar.config file > boot`. See https://quasar.dev/quasar-cli-vite/boot-files#usage-of-boot-files

**Reply by Alfredoarnoldo** [reply]
thank you for your answer!
It works fine now.
